conf/cassandra/migration/default/V1__create_portal.cql

```diff
@@ -41,4 +41,10 @@ PRIMARY KEY (organization, uuid);
 -- down:
 
 -- stage: 1
-DROP TABLE alerts;
\ No newline at end of file
+DROP MATERIALIZED VIEW alerts_by_organization;
```

Comment:
Modifying the v1 file doesn't seem safe.

Reply:
Agreed, it's certainly not good. But it's important to have the downs match the ups.